core/toolbox.js

```diff
@@ -520,6 +520,19 @@ Blockly.Toolbox.TreeControl.prototype.handleTouchEvent_ = function(e) {
  * @override
  */
 Blockly.Toolbox.TreeControl.prototype.createNode = function(opt_html) {
+    if (this.toolbox_ && !this.eventHandlerRegistered_) {
```

Comment:
Add a comment about this change, makes it easier when we have to merge with mainline blockly.